### PR TITLE
SD-52: add number format to stacking label

### PR DESCRIPTION
**Pull request by naibinh**

### CODE REVIEW

**Comment by naibinh** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/241/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/909/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/103/ : SUCCESS in 9m 50s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/254/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/934/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/109/ : SUCCESS in 10m 15s

File: src/components/visualizations/chart/highcharts/customConfiguration.ts [hunk 1]
```diff
@@ -788,7 +787,12 @@ export function getCustomizedConfiguration(chartOptions: IChartOptions, chartCon
         return merge(config, configurator(chartOptions, config));
     }, {});
 
-    const labelConfigData = getLabelsConfiguration(chartOptions, chartConfig);
+    const chartConfigData = [
```
Comment (ivan-nejezchleb) [inline src/components/visualizations/chart/highcharts/customConfiguration.ts]: what about to keep getLabelsConfiguration and getStackingConfiguration in array of configurators and only add third param:
`function getLabelsConfiguration(chartOptions: any, {}: any, config?: IChartConfig) {...`
and 
`return merge(config, configurator(chartOptions, config, chartConfig));`

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/942/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/485/ : SUCCESS in 1m 32s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/264/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/943/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/112/ : SUCCESS in 7m 58s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/192/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/488/ : SUCCESS in 1m 28s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/266/ : SUCCESS in 8m 32s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/85/ : SUCCESS in 8m 59s

**Pull request merged**